src/benchmarks/gc/README.md

```diff
@@ -29,65 +31,24 @@ On other systems it’s better to use your system’s package manager.
 py -m pip install -r src/requirements.txt
 ```
 
-You will likely run into trouble installing pythonnet.
-
-First, pythonnet is only needed to analyze test results, not to run tests.
-If you just want to run tests on this machine, you could comment out pythonnet from `src/requirements.txt`.
-Then when running tests, provide the `--no-check-runs` option.
-
-
-#### Pythonnet on Windows
-
-On Windows, if you run into trouble installing pythonnet, look for an error like:
-
-    Cannot find the specified version of msbuild: '14' 
-
-or:
-
-    Could not load file or assembly 'Microsoft.Build.Utilities, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
-
-If so, you may need to install Visual Studio 2015.
-
-
-#### Pythonnet on other systems
-
-Pythonnet [does not work](https://github.com/pythonnet/pythonnet/issues/939) with the latest version of mono, so you'll need to downgrade that to version 5.
-
-On Ubuntu the instructions are:
-
-* Change `/etc/apt/sources.list.d/mono-official-stable.list` to:
-```
-deb https://download.mono-project.com/repo/ubuntu stable-bionic/snapshots/5.20.1 main
-```
-* `sudo apt remove mono-complete`
-* `sudo apt update`
-* `sudo apt autoremove`
-* `sudo apt install mono-complete`
-* `mono --version`, should be 5.20.1
-
-Then to install from source:
 
-* Instructions: https://github.com/pythonnet/pythonnet/wiki/Installation
-* `py setup.py bdist_wheel --xplat`
-* WARN: The instructions there tell you to run `pip install --no-index --find-links=.\dist\ pythonnet`.
-  This may "succeed" saying `Requirement already satisfied: pythonnet in /path/to/pythonnet`.
-  INSTEAD, go to the *parent* directory and use `sudo python3.7 -m pip install --no-index --find-links=./pythonnet/dist/` which circumvents this bug.
-* Run `import clr` in the python interpreter to verify that installation worked.
+### Install pythonnet
 
+Pythonnet is only needed to analyze test results, not to run tests.
+If you just want to run tests on this machine, you can skip installing pythonnet,
+copy bench output to a different machine, and do analysis there.
+When running tests, you must provide the `--no-check-runs` option to avoid using pythonnet.
```

Comment:
if Pythonnet is not needed anyway, why does one need to specify anything to avoid using it? sounds confusing.

Reply:
There was code `_check_test_run` for verifying that a test run had the number of heaps expected; this hasn't caught any bugs in a very long itmes so I've simply removed it and you no longer have to pass `--no-check-runs`.